src/CharsetConverter.php

```diff
@@ -94,7 +94,7 @@ protected static function filterEncoding(string $encoding): string
             return $encoding_list[$key];
         }
 
-        throw new OutOfRangeException('The submitted charset '.$encoding.' is not supported by the mbstring extension.');
+        throw new OutOfRangeException('The submitted charset ' . $encoding . ' is not supported by the mbstring extension.');
```

Comment:
Why adding spaces ? Did you run `composer phpcs` ?

Reply:
Well, I've checked standards and haven't find there any consistency about spaces around concatenation dot.
My point is readability matters.
Anyway, I think we don't need my fix here :)